### PR TITLE
Add layer scoped actions in Attribute Table

### LINKED ISSUE #1523: [feature] Add layer scoped actions
Original commit: https://github.com/qgis/QGIS/commit/5b4a88f8e07e8ee1f9bfc0570f76f68717688cf2 by m-kuhn

Within the attribute table, there is a new button to trigger actions
which are not based on individual features but instead on the whole layer.
Normally they will perform actions based on all features or the selection.

In addition to this, a lot of cleanup has been done.

### PR DESCRIPTION
**Pull request by yjacolin**
Fix #1523

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -2661,6 +2661,11 @@ on **[Create default actions]**. To edit any of the examples, double-click
 its row. One example is performing a search based on an attribute value. This
 concept is used in the following discussion.
 
+The |checkbox| :guilabel:`Show in Attribute Table` enable a settings to add
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: enables a setting

**Comment by SrNetoChan** [comment]
@yjacolin and @DelazJ any reason for not merging this? maybe Yveswant to review, or maintain the description?

BTW, this would not be the kind of direct changes I was proposing on the mailing list, but just small fixes in grammar or typos.

**Comment by DelazJ** [comment]
> @yjacolin and @DelazJ any reason for not merging this? maybe Yveswant to review, or maintain the description?

Waiting for Yves agreement...
> BTW, this would not be the kind of direct changes I was proposing on the mailing list, but just small fixes in grammar or typos.

Yeah! My changes were expected at first to address typos I commented a month ago and then I realised that the description was actually not right so fixing only the typos would have been a loss of time imho.
And as I was mentioning in #2895 the Actions tab description needs rather a deep review due to 3.0 refactoring.

**Comment by yjacolin** [comment]
I stopped to work on this PR after seeing #2895 in order to not work on the same feature (note that I have already worked on @DelazJ feedback but not push it on the branch yet before #2895).

**Comment by SrNetoChan** [comment]
Regarding this PR, I think I would just add a link to the actions description that was updated in #2895.

**Comment by DelazJ** [comment]
@SrNetoChan I don't understand what you mean. What do you want to link to the actions description (which btw is 3 paragraphs below)?
To avoid further discussion on this PR, I will just merge and let you PR the requested changes.

**Pull request merged**